modules/ROOT/partials/configuration/table_merge_content_on_paste.adoc

```diff
@@ -0,0 +1,52 @@
+[[table_merge_content_on_paste]]
+== `+table_merge_content_on_paste+`
+
+include::partials$misc/admon-requires-6.5v.adoc[]
```

Comment:
@brian-forte Do we need to bump the version here to be `6.5.1`?

Reply:
The new option would exist on 6.5 so shouldn't be necessary.